news/2021-11-27-new-featured-artist-marmalade-butcher.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "New Featured Artist: Marmalade butcher"
+date: 2021-11-27 09:00:00 +0000
+---
+
+Our newest Featured Artist **Marmalade butcher** hits you in the face with their innovative math rock sound!
+
+![](https://assets.ppy.sh/artists/219/header.jpg)
+
+Blending some of music's less conventional genres into their own soup of aural magic, **Marmalade butcher** is a next-generation hybrid band that took flight in 2010. Renowned for their complex sound that drifts from the usual electronic fare from rhythm games, this group is the ideal change for anyone and everyone in the osu! mapping scene.
+
+We've secured **23** jaw-dropping tracks (all of which can be found on [**Marmalade butcher**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/220)), including a couple that avid rhythm gamers might recognize from *pop'n music peace*!
+
+**Marmalade butcher** may have only received their first ranked map a few months ago, but their presence in osu! has been ever-present. Take a peek at the intensity of their tracks through a video of [*Marmalade butcher - Y/M/K/M*](https://osu.ppy.sh/beatmapsets/1502963), then try out any of the other awesome beatmaps further below:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/220/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Marmalade butcher - Y/M/K/M
+
+Experience [the beatmap from the video above](https://osu.ppy.sh/beatmapsets/1502963) hosted by [Lasse](https://osu.ppy.sh/users/896613)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Marmalade butcher - Ushinahareru Runazuki
+
+See how **Marmalade butcher** translates to osu!mania through [this beatmap](https://osu.ppy.sh/beatmapsets/1486395) hosted by [Shoegazer](https://osu.ppy.sh/users/2520707)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Marmalade butcher - Amanita
+
+Check out [this popular graveyard map](https://osu.ppy.sh/beatmapsets/1139182) hosted by [xbopost](https://osu.ppy.sh/users/6842421) or [this map from OWC 2020](https://osu.ppy.sh/beatmapsets/1306572) created by [DeviousPanda](https://osu.ppy.sh/users/4966334), [Mao](https://osu.ppy.sh/users/2204515), and [Zelq](https://osu.ppy.sh/users/8953955)!
```

Comment:
iirc Panda's Amanita have been moved to http://osu.ppy.sh/beatmapsets/1473778#osu/3079004 so i think it would be more accurate to phrase this as something like this :

`...or [this map](https://osu.ppy.sh/beatmapsets/1473778) hosted by [Camo](https://osu.ppy.sh/users/5194391) which contains the distinguished OWC 2020 difficulty made by [DeviousPanda](https://osu.ppy.sh/users/4966334), [Mao](https://osu.ppy.sh/users/2204515), and [Zelq](https://osu.ppy.sh/users/8953955)!`

Reply:
did it, but made it a bit easier to read